src/emails/templates/monthlyActivity/freeUser/MonthlyActivityFreeUser.tsx

```diff
@@ -0,0 +1,290 @@
+import React from "react";
+import type { SubscriberRow } from "knex/types/tables";
+import { ExtendedReactLocalization } from "../../../../app/functions/l10n";
+import { EmailFooter } from "../../EmailFooter";
+import { EmailHeader } from "../../EmailHeader";
+import { HibpLikeDbBreach } from "../../../../utils/hibp";
+import { FeatureFlagName } from "../../../../db/tables/featureFlags";
+import type { LatestOnerepScanData } from "../../../../db/tables/onerep_scans";
+import type { SubscriberBreach } from "../../../../utils/subscriberBreaches";
+import { EmailHero } from "../../../components/EmailHero";
+import { DataPointCount } from "../../../components/EmailDataPointCount";
+import { DashboardSummary } from "../../../../app/functions/server/dashboard";
+import { EmailBanner } from "../../../components/EmailBanner";
+import { getPremiumSubscriptionUrl } from "../../../../app/functions/server/getPremiumSubscriptionInfo";
+import { isEligibleForPremium } from "../../../../app/functions/universal/premium";
+import { getSignupLocaleCountry } from "../../../functions/getSignupLocaleCountry";
+
+export type MonthlyReportFreeUserEmailProps = {
+  l10n: ExtendedReactLocalization;
+  breach: HibpLikeDbBreach;
+  breachedEmail: string;
+  utmCampaignId: string;
+  utmContentSuffix: string;
+  subscriber: SubscriberRow;
+  month: string;
+  dataSummary: DashboardSummary;
+  unsubscribeLink: string;
+};
+
+export const MonthlyReportFreeUserEmail = (
+  props: MonthlyReportFreeUserEmailProps,
+) => {
+  const l10n = props.l10n;
+
+  const premiumSubscriptionUrlObject = new URL(
+    getPremiumSubscriptionUrl({ type: "yearly" }),
+  );
+  const assumedCountryCode = getSignupLocaleCountry(props.subscriber);
+  const hasRunFreeScan = typeof props.subscriber.onerep_profile_id === "number";
+
+  const bannerDataCta = {
+    label: hasRunFreeScan
+      ? l10n.getString("email-monthly-report-free-banner-cta-upgrade")
+      : l10n.getString("email-monthly-report-free-banner-cta-free-scan"),
+    link: hasRunFreeScan
+      ? premiumSubscriptionUrlObject.href
+      : `${process.env.SERVER_URL}/user/dashboard/?utm_source=monitor-product&utm_medium=email&utm_campaign=${props.utmCampaignId}&utm_content=take-action${props.utmContentSuffix}`,
+  };
+
+  const purpleActiveColor = "#7542E5";
+  const greyInactiveColor = "#9E9E9E";
+
+  const resolvedBoxData = {
+    dataPointCountLabel: hasRunFreeScan
+      ? "email-monthly-report-free-summary-manually-resolved-exposures"
+      : "email-monthly-report-free-summary-resolved-breaches",
+    dataPointValue: hasRunFreeScan
+      ? props.dataSummary.dataBreachResolvedNum +
+        props.dataSummary.dataBrokerManuallyResolvedNum
+      : props.dataSummary.dataBreachResolvedNum,
+    activeState:
+      (hasRunFreeScan &&
+        props.dataSummary.dataBreachResolvedNum +
+          props.dataSummary.dataBrokerManuallyResolvedNum >
+          0) ||
+      (!hasRunFreeScan && props.dataSummary.dataBreachResolvedNum > 0),
```

Comment:
Yeah but I mean, for the calculation of the numbers, if the user hasn't run a scan, we should still be able to just add `dataBrokerManuallyResolvedNum` to the total, since it'll be 0, right? Or in other words, a change like this should have the same result?

```diff
-      (hasRunFreeScan &&
-        props.dataSummary.dataBreachResolvedNum +
-          props.dataSummary.dataBrokerManuallyResolvedNum >
-          0) ||
-      (!hasRunFreeScan && props.dataSummary.dataBreachResolvedNum > 0),
+      props.dataSummary.dataBreachResolvedNum +
+        props.dataSummary.dataBrokerManuallyResolvedNum >
+        0
```